docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,659 @@
+  | * read-event-by-id                                                                  |
+  | * subscribe-event-by-id                                                             |
+  +-------------------------------------------------------------------------------------+
+[1647417645.183836][404411:404411] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/Commands.cpp:84: Error 0x0000002F
+
+```
+### Get the list of attributes supported for a specific cluster
+```
+./BUILD_PATH/chip-tool <cluster_name> read
+```
+where:
+-   *<cluster_name\>* is one of the available clusters (listed in previous section)
+
+Example:
+```
+./BUILD_PATH/chip-tool onoff read
+```
+Output:
+```bash
+[1647417857.913942][404444:404444] CHIP:TOO: Missing attribute name
+Usage:
+  ./chip-tool onoff read attribute-name [param1 param2 ...]
+
+  +-------------------------------------------------------------------------------------+
+  | Attributes:                                                                         |
+  +-------------------------------------------------------------------------------------+
+  | * on-off                                                                            |
+  | * global-scene-control                                                              |
+  | * on-time                                                                           |
+  | * off-wait-time                                                                     |
+  | * start-up-on-off                                                                   |
+  | * server-generated-command-list                                                     |
+  | * client-generated-command-list                                                     |
+  | * attribute-list                                                                    |
+  | * feature-map                                                                       |
+  | * cluster-revision                                                                  |
+  +-------------------------------------------------------------------------------------+
+[1647417857.914110][404444:404444] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/Commands.cpp:120: Error 0x0000002F
+```
+### Get the list of command options
+
+To get the list of parameters for a specific command, run the `chip-tool` executable with the target cluster name and the target command name
+
+Example:
+```
+./BUILD_PATH/chip-tool onoff on
+```
+Output:
+```bash
+[1647417976.556313][404456:404456] CHIP:TOO: InitArgs: Wrong arguments number: 0 instead of 2
+Usage:
+  ./chip-tool onoff on node-id/group-id endpoint-id-ignored-for-group-commands [--paa-trust-store-path] [--commissioner-name] [--trace_file] [--trace_log] [--ble-adapter] [--timedInteractionTimeoutMs] [--suppressResponse]
+[1647417976.556362][404456:404456] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/Commands.cpp:135: Error 0x0000002F
```

Comment:
I'm not sure if I would provide this error log. It is not giving a user any useful information and code may change in time, so the documentation will be out of date then.

Reply:
I considered that as well and decided to leave errors from chip-tool. We can change this approach, but let's imagine that errors are not listed here and the user runs suggested commands and suddenly there are errors all over the place... that just might be confusing. We may also just mention somewhere that chip-tool reports these kind of stuff, although it's rather false positive.

Reply:
Yeah it's kind of true. Maybe I just used to the fact that closing controller ends always with segfault, so printed errors doesn't bother me  :smile: